apps/client/src/app/services/data.service.ts

```diff
@@ -150,6 +150,10 @@ export class DataService {
     return this.http.delete<any>(`/api/v1/order/${aId}`);
   }
 
+  public deleteAllOrders() {
```

Comment:
Can you move this function up so the order is alphabetical?

Reply:
there is some kind of plugin that does this automatically :) maybe worth investigating